releasenotes/notes/remove-deprecated-code-in-0.22.0-139fa09ee0cc6df9.yaml

```diff
@@ -0,0 +1,36 @@
+
+upgrade:
+
+  - |
+    Removed the argument `qubit_channel_mapping` in :class:`qiskit.transpiler.passes.calibration.rzx_builder.RZXCalibrationBuilder`
+    
+    old:
+   	  # Create an RZXCalibrationBuilder object with the qubit_channel_mapping argument 
+      builder = RZXCalibrationBuilder(qubit_channel_mapping=[[0, 1]])
+    new:
+      # Create a target object for the Aer simulator
+      target = providers.aer.QasmSimulator().target()
+      # Create an RZXCalibrationBuilder object using the target object
+      builder = RZXCalibrationBuilder(target=target)
```

Comment:
I believe that not specifying these lines as code blocks is confusing Sphinx and breaking the lint and docs build (see the [docs workflow](https://dev.azure.com/qiskit-ci/qiskit/_build/results?buildId=49924&view=logs&j=abfb3e93-d1ff-5b1b-cb4c-74d6dcf536f9&t=60edc4b3-4259-59c8-ae13-dc31febe2d3d&l=254)). While you can add code blocks with the following syntax:
```
    Here is some reno text... For example:

    .. code-block:: python
    
      # This is a comment
      # Respecting the indentation of the code block is important
      x = 1
```
I think that this example may actually confuse users, as it looks like `target` is a direct replacement for `qubit_channel_mapping`, when this not the case. Given that this was a deprecation without replacement, I'd say that we don't need a migration example here. I would directly remove it and keep it simple.

Reply:
Thank you so much for the feedback. Let me update the .yaml file and push the commit